src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,294 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Interfaces;
+using Colors.Net;
+using DurableTask.AzureStorage;
+using DurableTask.Core;
+using DurableTask.Core.History;
+using Microsoft.Azure.WebJobs.Script;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Colors.Net.StringStaticMethods;
+
+namespace Azure.Functions.Cli.Common
+{
+    internal class DurableManager : IDurableManager
+    {
+        private readonly AzureStorageOrchestrationService _orchestrationService;
+
+        private readonly TaskHubClient _client;
+
+        private string TaskHubName;
+
+        private string ConnectionString;
+      
+        public DurableManager(ISecretsManager secretsManager)
+        {
+            this.SetConnectionStringAndTaskHubName(secretsManager);
+
+            var settings = new AzureStorageOrchestrationServiceSettings
+            {
+                TaskHubName = TaskHubName,
+                StorageConnectionString = ConnectionString,
+            };
+
+            _orchestrationService = new AzureStorageOrchestrationService(settings);
+            _client = new TaskHubClient(_orchestrationService);
+        }
+
+        // HELP WANTED: Is there a better way to do this?? (parse host.json for durable settings)
```

Comment:
@ahmelsayed is Glenna's approach for parsing host.json below the right thing to do?